rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/450_time_series.yml

```diff
@@ -0,0 +1,80 @@
+setup:
+  - skip:
+        version: " - 8.1.0"
+        reason: Suipport for time_series aggs was added in 8.1.0
+
+  - do:
+      indices.create:
+        index: tsdb
+        body:
+          settings:
+            number_of_replicas: 0
+            mode: time_series
+            routing_path: [key]
+            time_series:
+              start_time: "2021-01-01T00:00:00Z"
+              end_time: "2022-01-01T00:00:00Z"
+          mappings:
+            properties:
+              key:
+                type: keyword
+                time_series_dimension: true
+              "@timestamp":
+                type: date
+
+  - do:
+      cluster.health:
+        wait_for_status: green
+
+  - do:
+      bulk:
+        index: tsdb
+        refresh: true
+        body:
+          - '{ "index": {} }'
+          - '{ "key": "bar", "val": 2, "@timestamp": "2021-01-01T00:00:10Z" }'
+          - '{ "index": {}}'
+          - '{ "key": "bar", "val": 10, "@timestamp": "2021-01-01T00:00:00Z" }'
+          - '{ "index": {}}'
+          - '{ "key": "bar", "val": 50, "@timestamp": "2021-01-01T00:00:30Z" }'
+          - '{ "index": {}}'
+          - '{ "key": "bar", "val": 40, "@timestamp": "2021-01-01T00:00:20Z" }'
+
+  - do:
+      bulk:
+        index: tsdb
+        refresh: true
+        body:
+          - '{ "index": {} }'
+          - '{ "key": "foo", "val": 20, "@timestamp": "2021-01-01T00:00:00Z" }'
+          - '{ "create": {} }'
+          - '{ "key": "foo", "val": 30, "@timestamp": "2021-01-01T00:10:00Z" }'
+          - '{ "index": {} }'
+          - '{ "key": "baz", "val": 20, "@timestamp": "2021-01-01T00:00:00Z" }'
+          - '{ "index": {} }'
+          - '{ "key": "baz", "val": 20, "@timestamp": "2021-01-01T00:00:00" }'
```

Comment:
Maybe worth a comment. I think this is an effort to maybe have there be two segments.